app/models/download_acceptance.rb

```diff
@@ -0,0 +1,28 @@
+class DownloadAcceptance
+  include Mongoid::Document
+  include Mongoid::Timestamps
+
+  belongs_to :download_agreement
+  field :email, type: String
```

Comment:
I assume the reason you track email rather than user is so we still have a record even if the user is deleted?  If so, add a comment here explaining why we track email instead of user_id

Reply:
Yes, that is the case - it's also the same thing we do in `TosAcceptance`.

Reply:
2e76f7c